src/libslic3r/GCode.cpp

```diff
@@ -1395,7 +1395,7 @@ void GCode::_do_export(Print& print, GCodeOutputStream &file, ThumbnailsGenerato
         file.write_format(
             "; %s\n\n",
             std::string(
-                std::string("generated by SuperSlicer " SLIC3R_VERSION " on ") +
```

Comment:
I added this fake slicer ID so that Klipper's ECO system can recognize it.
I have added supports in Moonraker but there are other plugins(like the cancel objects script that need to be fixed).
Until then we need this fake slicer ID.

-SoftFever

Reply:
Thats fine it was only for my test